tibco/src/main/java/nl/nn/adapterframework/extensions/tibco/SendTibcoMessage.java

```diff
@@ -93,6 +97,16 @@ public void configure() throws ConfigurationException {
 			ConfigurationWarnings.add(this, log, "parameter [userName] has been replaced with [username]");
 		}
 
+		if(StringUtils.isNotEmpty(emsPropertiesFile)) {
```

Comment:
Dit is hetzelfde blok als in `GetTibcoQueues`. Kan dit misschien een method worden in `TibcoUtils`?

Reply:
De tool om een java properties bestand in te lezen zou een ConfiguratieExceptie kunnen gooien maar is dat niet foei-lelijk?